### ISSUE TITLE
[Adaptive Cards-ActivityUpdate.json]: Screen reader is not announcing descriptive information for <"Next"/ "Previous"> icons button.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9244]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen reader: Narrator/NVDA

**Repro steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to ActivityUpdate.json button by using Tab key and activate it.
3. ActivityUpdate.json adaptive card will be appeared on right sight of the pane.
4. Navigate to 'Set due date' control using tab key and activate it using enter key.
5. Navigate to the 'Enter the due date' calendar filed using tab key and activate using enter key.
6. Calendar pop up appears navigate to the <"Next"/ "Previous"> icons button using tab key.
7. Observe the issue.

**Actual Result:**
Screen reader is not announcing descriptive information for <"Next"/ "Previous"> icons button.

**Note**
Same issue replicates of the <"Next"/ "Previous"> icons button for the 'Due Date' calendar field in the 'InputsWithValidation.Json' card.

**Expected Result:**
Screen reader should announce information like as "Next month"/"Previous month" button.

**User Impact:**
Screen reader users will find it difficult to interact with the UI if no description is provided.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/068ffccf-56e4-4a35-b943-16e04f8ffede

![Image](https://github.com/user-attachments/assets/ba5eecb3-ed20-483c-b9ad-b7e619198a98)

### Card JSON

```JSON
N/A
```
